hoot-core/src/main/cpp/hoot/core/io/OsmGbdxJsonWriter.h

```diff
@@ -89,7 +89,7 @@ class OsmGbdxJsonWriter : public OsmJsonWriter
    * @brief _writeNodes Iterates all nodes that aren't part of another element and writes
    *   them out individual GeoJSON files
    */
-  void _writeNodes();
+  virtual void _writeNodes();
```

Comment:
`_writeNodes` exists in the base class also so it is overridden in some instances and can cause some errors.  Making it `virtual` solves any potential issues with this.